README.md

```diff
@@ -1,43 +1,71 @@
-# onnx-cpu
+# `onnx-cpu` modular resource
 
-## Build it
+This module allows you to run ONNX models with the Viam Vision Service on Android.
 
-if you need to package it up in a module, do the following
+## Requirements
 
-```
-go build -o module cmd/module/main.go
-chmod a+x module
-tar -czf module.tar.gz module third_party/
-```
+Before configuring your ML model, you must [create a machine](https://docs.viam.com/fleet/machines/#add-a-new-machine).
 
-`third_party` contains all of the `.so` files for the ONNX runntime. You can package only the one you need.
+## Build and run
 
-For Android, the Makefile will create the tar file.
+To use this module, follow these instructions to [add a module from the Viam Registry](https://docs.viam.com/registry/configure/#add-a-modular-resource-from-the-viam-registry) and select the `viam-labs:mlmodel:onnx-cpu` model from the [`onnx-cpu` module](https://github.com/viam-labs/onnx-cpu).
 
-You can then add the module to app.viam.com locally. the triplet is `viam-labs:mlmodel:onnx-cpu`
+## Configure your `onnx-cpu` sensor
```

Comment:
So wait-- this is a sensor? That is not clear from intro. Or is it an mlmodel service? seems like service but confusing name